components/eamxx/src/share/io/scorpio_output.cpp

```diff
@@ -53,7 +53,7 @@ AtmosphereOutput (const ekat::Comm& comm, const ekat::ParameterList& params,
       "Error! Unsupported averaging type '" + avg_type + "'.\n"
       "       Valid options: Instant, Max, Min, Average. Case insensitive.\n");
 
-  set_field_manager (field_mgr);
+  set_field_manager (field_mgr,"both");
```

Comment:
Assuming my comment inside `set_diagnostic` is correct, I believe we don't need the "both" mode. I believe we only need to set the sim FM at the beginning, since that's what we need to create diagnostics and, if needed, the remapper.